### PR TITLE
[Console] Add documentation for Console Negatable option

### LINKED ISSUE #14783: [Console] Support binary / negatable options
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#39642
| PR author(s) | @greg-1-anderson, @jderusse
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by jderusse**
Fixes #14783

### CODE REVIEW

File: console/input.rst [hunk 1]
```diff
@@ -216,6 +216,14 @@ There are four option variants you can use:
     This option may or may not have a value (e.g. ``--yell`` or
     ``--yell=loud``).
 
+``InputOption::VALUE_NEGATABLE``
+    Accept either the flag (e.g. ``--yell``) or it negation (e.g.
```
Comment (OskarStark) [inline console/input.rst]:
```suggestion
    Accept either the flag (e.g. ``--yell``) or its negation (e.g.
```

File: console/input.rst [hunk 2]
```diff
@@ -216,6 +216,14 @@ There are four option variants you can use:
...
+    ``--no-yell``).
+
+.. versionadded:: 5.3
+
+    The ``InputOption::VALUE_NEGATABLE`` constant was introduced in Symfony 5.3
```
Comment (OskarStark) [inline console/input.rst]:
```suggestion
    The ``InputOption::VALUE_NEGATABLE`` constant was introduced in Symfony 5.3.
```

**Review by OskarStark — APPROVED** [review]

File: console/input.rst (hunk 2 above)
Comment (jderusse) [reply console/input.rst]: thank you

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Jérémy for working on this feature, this is much appreciated.

**Pull request merged**